README.md

```diff
@@ -101,20 +101,34 @@ For example:
 
 You can find much more detail in the [create-react-app documentation][].
 
-## Other Tools For Creating React Apps
+## Create React App Alternatives
 
-While Create React App is a great choice for your projects at Flatiron, there
-are some excellent alternatives that might make sense for other projects you
-build later on. The [React Docs][react-docs-create] list a few of the most
-popular ones. A couple of highlights:
+Although the new React docs recommend frameworks like Next.js and Remix, we
+recommend that you continue using Create React App since those frameworks
+require you to already know React fundamentals and API development principles.
+Once you’ve completed Phase 4, you will have the fundamental concepts required
+to learn new frameworks.
 
-- **Gatsby**: a static-site generator that turns your application into separate
-  pages by building HTML files from JSX components. It's a popular choice for
-  personal blogs, including Dan Abramov's
-  [overreacted][]
-- **Next.js**: a framework for creating static and server-side generated React
-  applications. Read more about the differences between Client-Side Rendering
-  and Server-Side Rendering [here][csr-ssr].
+If you want to use a tool that is similar to CRA after you’ve completed our
+course, we recommend using [Vite](https://vitejs.dev/). It manages the
+complexities of setting up a React app just like CRA while adding features like
+universal plugins, configs for vanilla setups or other frameworks, hot module
+replacement.
+
+## When to Use Meta-Frameworks?
+
+Frameworks like Next.js and Remix are called meta-frameworks since they are
+built on top of an existing framework like React.
+
+Next.js is likely the most popular one at the moment. It provides several
+advanced features such as built-in routing, different rendering techniques based
+on routing, data fetching systems. All of this can make development a lot
+easier. But in order to understand and use these features properly, you need to
+understand the core principles of both front end and back end development.
+
+This is why we recommend students try these out after graduating from the
+program or for their capstone project if they feel comfortable with configuring
+these tools.
```

Comment:
Recommend replacing the above paragraph with something like this:

"For this reason, we recommend students wait to try out these tools until after they have completed the program and have a solid understanding of the underlying principles."

I'm not sure about including the capstone option - I feel like pretty much all students would be better off not going down that road. If you disagree, though, I'm okay with leaving it in.